lang/en_us/format_topcoll.php

```diff
@@ -67,3 +67,10 @@
 // Capabilities.
 $string['topcoll:changecolour'] = 'Change or reset the color';
 
+// Toggle Display Blocks
```

Comment:
If the language strings are identical to 'en' then this is not needed.

Reply:
No longer need: https://github.com/gjb2048/moodle-format_topcoll/pull/98/files#diff-c0088f30815fb5f4137495566dc9e7b9695331b1d54e3338fb13677ab4c55e85R70.